aws/src/integration/java/org/apache/iceberg/aws/glue/TestGlueCatalogTable.java

```diff
@@ -206,6 +208,15 @@ public void testUpdateTable() {
         .isEqualTo("EXTERNAL_TABLE");
     assertThat(response.table().storageDescriptor().columns()).hasSameSizeAs(schema.columns());
     assertThat(response.table().partitionKeys()).hasSameSizeAs(partitionSpec.fields());
+    assertThat(response.table().description()).isEqualTo(description);
+
+    // Make sure adding a comment updates the existing table description
+    String comment = "Test comment";
```

Comment:
Nit: I think if you rename the variable you can probably get rid of the inline comment since it'll be clear.

Maybe just call it `updatedDescription` or `updatedComment` and then you won't need the inline comment since the test is clear.

Reply:
Make sense.